MIGRATION.md

```diff
@@ -123,6 +123,19 @@ For our efforts to reduce bundle size of the SDK we had to remove and refactor p
   `setupBrowserTransport` or `setupNodeTransport` for default transports, depending on your requirements.
 - Remove support for Opera browser pre v15
 
+## Sentry Angular SDK Changes
+
+The Sentry Angular SDK (`@sentry/angular`) is now compiled with the Angular compiler (see [#4641](https://github.com/getsentry/sentry-javascript/pull/4641)). This change was necessary to fix a long-lasting bug in the SDK (see [#3282](https://github.com/getsentry/sentry-javascript/issues/3282)): `TraceDirective` and `TraceModule` can now be used again without risking an application compiler error or having to disable AOT compilation.
+
+### Angular Version Compatibility
+
+Given the forward compatibility of the Angular compiler, v7 of our SDK will only work with Angular 10 and above. Previously, it was possible to use the SDK with versions <10, although we officially only supported Angular 10-13 as `peerDependencies`. If you are using Angular <10 in your project, we recommend staying on the latest 6.x version until you can upgrade your Angular version.
```

Comment:
This is totally optional. Suggesting because my brain kinda froze while reading that sentence because of the code block.

```suggestion
Given the forward compatibility of the Angular compiler, v7 of our SDK will only work with Angular 10 and above. Previously, it was possible to use the SDK with versions <10, although we officially only supported Angular 10-13 as peer dependencies. If you are using Angular <10 in your project, we recommend staying on the latest 6.x version until you can upgrade your Angular version.
```

Reply:
Makes sense, good brain reaction ;)